extensions/ql-vscode/src/data-extensions-editor/auto-model.ts

```diff
@@ -68,6 +68,61 @@ export function createAutoModelRequest(
   return request;
 }
 
+export function parsePredictedClassifications(
```

Comment:
Non-blocking: It might be nice putting a comment saying something along the lines of what I wrote here: https://github.com/github/vscode-codeql/pull/2448#discussion_r1206645139